src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -126,6 +126,17 @@ public class Robot {
 
     private DirectColorModel screenCapCM = null;
 
+    /**
+     * Default 20 milliseconds delay for mouse {@code click} and
+     * step delay for mouse {@code glide}.
+     */
+    public static final int DEFAULT_DELAY = 20;
+
+    /**
+     * Default 2 pixel step length for mouse {@code glide}.
+     */
+    public static final int DEFAULT_STEP_LENGTH = 2;
```

Comment:
Do we want to make the `DEFAULT_DELAY` and `DEFAULT_STEP_LENGTH` configurable?

Reply:
glide  has over-rides step length

Reply:
I think setting the default step length with a setter/getter makes sense. I think generally tests don't change the step delays across their glides anyway and it would remove the need for all those extra overrides for glide. I'll make this change now